src/styles/blocks/stars.css

```diff
@@ -0,0 +1,29 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('../../images/star.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+
+  height: 12px;
+  width: 12px;
+
+  margin: 2px;
+}
+
+/* this margin is used instead `gap` property */
+.stars__star:not(:last-child) {
+  margin-right: 6;
```

Comment:
The `margin-right` value is missing a unit. You should specify the unit (e.g., `px`), to ensure consistent rendering across browsers.